hls/lib/hls_endpoint.ex

```diff
@@ -121,16 +122,23 @@ defmodule Membrane.RTC.Engine.Endpoint.HLS do
   end
 
   @impl true
-  def handle_init(_context, options) do
+  def handle_init(ctx, options) do
+    {:endpoint, endpoint_id} = ctx.name
+
+    subscriptions_state = %Subscriptions.State{
```

Comment:
I don't like the fact that we directly look into the internal state of the `Subscriber`. What do you think about invoking some `tracks` or `get_tracks` function instead?